From 50109d56c8357cd6f40515e38a6dd16e243fd085 Mon Sep 17 00:00:00 2001
From: Ubuntu <ubuntu@ip-172-27-31-70.ec2.internal>
Date: Wed, 11 Jul 2018 18:14:20 +0000
Subject: [PATCH] always ask password & black root

---
 data/lxdm.conf.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/data/lxdm.conf.in b/data/lxdm.conf.in
index 6206262..fd98a53 100644
--- a/data/lxdm.conf.in
+++ b/data/lxdm.conf.in
@@ -16,7 +16,7 @@
 # xauth_path=/tmp

 # not ask password for users who have empty password
-# skip_password=1
+# skip_password=0

 ## greeter used to welcome the user
 greeter=@libexecdir@/lxdm-greeter-gtk
@@ -58,5 +58,5 @@ disable=0
 white=

 ## blacklist user
-black=
+black=root

--
2.7.4